sauce/features/budget/budget-progress-bars/index.js

```diff
@@ -0,0 +1,229 @@
+import { Feature } from 'toolkit/core/feature';
+import { getCurrentRouteName, getEntityManager } from 'toolkit/helpers/toolkit';
+
+export class BudgetProgressBars extends Feature {
+  // Supporting functions, or variables, etc
+  loadCategories = true;
+  selMonth;
+  subCats = [];
+  internalIdBase;
+  monthProgress;
+  progressIndicatorWidth = 0.005; // Current month progress indicator width
```

Comment:
Since this one is a constant it'd be good to chuck it up above the class as `const progressIndicatorWidth = 0.005;` so it's clear that it's not a class variable. You then don't need to use `this` to access it as well.

Reply:
Done.